src/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.Buffer.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System.Diagnostics;
```

Comment:
@KrzysztofCwalina, which name is more suitable in this case: `BigIntegerCalculator.BitsBuffer.cs` or `BitsBuffer.cs`?